forge-gui/res/cardsfolder/upcoming/throne_of_eldraine.txt

```diff
@@ -0,0 +1,8 @@
+Name:Throne of Eldraine
+ManaCost:5
+Types:Legendary Artifact
+K:ETBReplacement:Other:ChooseColor
+SVar:ChooseColor:DB$ ChooseColor | Defined$ You | AILogic$ MostProminentInComputerDeck | SpellDescription$ As CARDNAME enters the battlefield, choose a color.
+A:AB$ Mana | Cost$ T | Produced$ Chosen | Amount$ 4 | RestrictValid$ Spell.ChosenColor+MonoColor | SpellDescription$ Add four mana of the chosen color. Spend this mana only to cast monocolored spells of that color.
+A:AB$ Draw | Cost$ Mana<3\ChosenColor> T | NumCards$ 2 | SpellDescription$ Draw two cards. Spend only mana of the chosen color to activate this ability.
```

Comment:
@tool4ever don't know where in ComputerUtilMana to filter this like the XColor param, but maybe this is possible?...

> **Cost$ 3 T | ManaPref$ ChosenColor**

Reply:
Well it's not only a Pref, other colors aren't allowed so I need to do more support